mindjunkies/home/tests/test_enrollment_tags.py

```
@@ -0,0 +1,75 @@
+        """Test that the filter handles None as course gracefully"""
+        result = get_enrollment(None, self.user)
+        self.assertIsNone(result)
+
+    def test_get_enrollment_with_none_user(self):
+        """Test that the filter handles None as user gracefully"""
+        result = get_enrollment(self.course, None)
+        self.assertIsNone(result)
+
+    def test_template_usage(self):
+        """Test the filter in a template context"""
+        context = template.Context({
```

Ruff (0.8.2)

Local variable `context` is assigned to but never used

Remove assignment to unused variable `context`

(F841)